test/test_facevalues.jl

```diff
@@ -199,3 +199,38 @@
     end
 
 end # of testset
+
+@testset "EmbeddedLineFacetValues" begin
+
+    for dim in (2, 3)
+        for (order, ct) in zip((1, 2), (Line, QuadraticLine))
+            grid = generate_grid(ct, (1,), ones(Vec{dim}), dim == 2 ? Vec((4.0, 3.0)) : Vec((4.0, 3.0, 5.0)))
+            ip = Lagrange{RefLine, order}()
+            ipGeo = Lagrange{RefLine, order}()^dim
+
+            qr = QuadratureRule{RefLine}(2)
+            cv = CellValues(qr, ip, ipGeo)
+            cc = CellCache(grid)
+            Ferrite.reinit!(cc, 1)
+            Ferrite.reinit!(cv, cc)
+
+            fqr = FacetQuadratureRule{RefLine}(1)
+            fv = FacetValues(fqr, ip, ipGeo)
+            fc = FacetCache(grid)
+
+            x = getproperty.(getnodes(grid), :x)
+            dxdξ = Ferrite.calculate_mapping(cv.geo_mapping, 1, x).J
+
+            for i in 1:2
+                s = i == 1 ? -1 : 1
+                Ferrite.reinit!(fc, FaceIndex((1, i)))
+                Ferrite.reinit!(fv, fc)
+                @test first(fv.normals) ≈ Vec{dim}(s * dxdξ / norm(dxdξ))
+            end
+        end
+    end
+
+    # Test unknown facet error path as its not yet tested in "test_quadrules.jl"
+    err = ArgumentError("unknown facet number")
+    @test_throws err Ferrite.weighted_normal(zero(Ferrite.SMatrix{2, 1}), RefLine, 3)
```

Comment:
`SMatrix` should be available in the tests, if not add it to the `using StaticArrays` statement in runtests.jl. 

I suggest not testing the exact text in the error, since this typically is annoying if updating the text later. 

```suggestion
    @test_throws ArgumentError Ferrite.weighted_normal(zero(SMatrix{2, 1}), RefLine, 3)
```